### PR TITLE
Issue #1719 SqlDatabaseToolkit should have custom llm for QueryCheckerTool

### PR DESCRIPTION
**Pull request by rushout09**

### CODE REVIEW

**Review by sar009 — APPROVED** [review]

**Comment by tjaffri** [comment]
🚀 thank you, this will really help use smaller / local language models to generate SQL (since they make errors more often than OpenAI etc.)

**Review by hinthornw — COMMENTED** [review]
I like it

File: langchain/agents/agent_toolkits/sql/base.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def create_sql_agent(
     agent = ZeroShotAgent(llm_chain=llm_chain, allowed_tools=tool_names, **kwargs)
     return AgentExecutor.from_agent_and_tools(
         agent=agent,
-        tools=toolkit.get_tools(),
+        tools=tools,
```
Comment (agola11) [inline langchain/agents/agent_toolkits/sql/base.py:47]: ah, thanks for fixing this

**Review by agola11 — APPROVED** [review]
Thanks for this change!

**Comment by agola11** [comment]
can handle fixing CI checks, you'll still get attributed

**Comment by rushout09** [comment]
Thanks @agola11 . You guys are building some serious good stuff here. Looking forward to contribute more. My twitter: RushabhSAgarwal

**Pull request merged**